src/fu-tool.c

```diff
@@ -1568,10 +1568,6 @@ fu_util_update(FuUtilPrivate *priv, gchar **values, GError **error)
 	}
 
 	priv->current_operation = FU_UTIL_OPERATION_UPDATE;
-	g_signal_connect(FU_ENGINE(priv->engine),
```

Comment:
are you sure we don't still want to do this *after* we've got the device list?

Reply:
We already do it!  We do it in fu_util_install_release